src/main/java/de/hysky/skyblocker/utils/RomanNumerals.java

```diff
@@ -51,4 +61,24 @@ public static int romanToDecimal(String romanNumeral) {
 		}
 		return decimal;
 	}
+
+	/**
+	 * Converts a decimal number to a roman numeral.
+	 *
+	 * @param decimal The decimal number to convert.
+	 * @return The roman numeral, or an empty string if the number is out of range.
+	 */
+	public static String decimalToRoman(int decimal) {
```

Comment:
Could use a `@NotNull` for clarification since it doesn't return null
```suggestion
	@NotNull
	public static String decimalToRoman(int decimal) {
```

Reply:
And this requires an import smh
You can probably tell by now that I have a very strong dependency on my IDE to write code